vyxal/lexer.py

```diff
@@ -200,13 +200,9 @@ def tokenise(source: str) -> list[Token]:
                 contextual_token_value += source.popleft()
 
             if head == "→":
-                tokens.append(
-                    Token(TokenType.VARIABLE_SET, contextual_token_value)
-                )
+                tokens.append(Token(TokenType.VARIABLE_SET, contextual_token_value))
```

Comment:
is there some discrepancy in autoformatter here that's creating a lot of churn with re-formatting loads of code? We need to standardise the line length limit

Reply:
Honestly, I don't know. I'll see if I need to change black's settings or anything. It's probably just me being dumb, not black's fault.